packages/lsp-server/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "@xml-tools/xml-lsp-server",
+  "name": "@xml-tools/lsp-server",
   "description": "XML language server",
   "version": "1.0.0",
   "license": "Apache-2.0",
   "engines": {
     "node": ">=10.0.0"
   },
+  "main": "src/server.ts",
```

Comment:
- main is always a js file
- also I am not sure we want a require("@xml-tools/lsp-server) to automatically launch the server.
- I guess if we had a programmatic API to start the server it would appear here, but we do not yet.
- Perhaps our programmatic API should expose the **PATH** to the server file which is needed by the LSP Client?

Reply:
the path should be from the package published in npm.js no?

Reply:
- we could also expose the langauge services too (without the server)